pywhat/Data/regex.json

```diff
@@ -727,6 +727,19 @@
          "Credentials",
          "Password",
          "Username"
-       ]
-    }
-]
+      ]
+   },
+   {
+      "Name": "PGP Public Key",
+      "Regex": "/^(-----BEGIN PGP PUBLIC KEY BLOCK-----).*([a-zA-Z0-9//\n\/\\.\\:\\+\\ \\=]+).*(-----END PGP PUBLIC KEY BLOCK-----)$|^(-----BEGIN PGP PRIVATE KEY BLOCK-----).*([a-zA-Z0-9//\n\/\\.\\:\\+\\ \\=]+).*(-----END PGP PRIVATE KEY BLOCK-----)$/",
+      "plural_name": false,
+      "Description": null,
+      "Rarity": 0,

```

Comment:
Rarity is not 0 here, the regex is too specific :)

Reply:
Agree. @baldoarturo please change it to 1 and move the regex to the top of the file as we have it sorted by rarity. 😄